theandrewmo/hoppyhour
# Photos

## Pull Request: Adding Photo Upload for Reviews

### Description
This pull request introduces a new feature that allows users to upload photos along with their reviews. The uploaded photos are stored on Firebase, providing a seamless and reliable storage solution for user-generated content.

### Changes Made
- Created a new file upload functionality that integrates with Firebase Storage for handling photo uploads.
- Updated the review submission form to include a file input field for selecting and uploading photos.
- Implemented the necessary backend logic to handle the file upload, generate a Firebase Storage URL for each photo, and associate the URLs with the corresponding reviews.
- Added frontend validation to ensure that users can only upload image files.
- Improved the user interface to provide feedback during the photo upload process.

### How to Test
1. Navigate to the reviews section of the application.
2. Locate the form for submitting a new review.
3. Fill in the required review details (rating, description, etc.).
4. Choose an image file from your local machine using the newly added file input field.
5. Submit the review form.
6. Verify that the photo is successfully uploaded to Firebase and associated with the submitted review.

### Screenshots / Demo

https://github.com/theandrewmo/hoppyhour/assets/106348010/b9e39f46-b32b-4a62-8dcb-48d0e7a09c64


### Additional Notes
Thank you for reviewing this pull request. Your feedback and suggestions are greatly appreciated.
